components/esb-tools/plugins/org.wso2.integrationstudio.esb.form.editors/src/org/wso2/integrationstudio/esb/form/editors/article/providers/ValueExpressionWidgetNameSpacedPropertyEditorDialog.java

```diff
@@ -0,0 +1,1746 @@
+        Rectangle parentBounds = getParent().getBounds();
+        Rectangle dialogBounds = dialogShell.getBounds();
+        int centerX, centerY;
+        centerX = (parentBounds.width - dialogBounds.width) / 2 + parentBounds.x;
+        centerY = (parentBounds.height - dialogBounds.height) / 2 + parentBounds.y;
+        dialogShell.setLocation(new Point(centerX, centerY + 85)); //adding a constant number of pixels to avoid removing borders
+    }
+
+    private void setTabOrder() {
+        Control[] tabOrder = new Control[] { nsPrefixTextField, nsUriTextField, addButton };
+  
+        namespacesGroupBox.setTabList(tabOrder);
+
+        tabOrder = new Control[] { okButton };
+        dialogShell.setTabList(tabOrder);
+        
+        tabOrder = new Control[] { xmlContentGroupBox, treeViewGroupBox };
+        xmlContentTabFolder.setTabList(tabOrder);
+    }
+
+    private void saveConfiguration() throws Exception {
+//        TransactionalEditingDomain domain = TransactionUtil.getEditingDomain(nsProperty);
+//        domain.getCommandStack().execute(new RecordingCommand(domain) {
+//
+//            @Override
+//            protected void doExecute() {
+                nsProperty.setPropertyValue(expressionTextField.getText());
+                nsProperty.getNamespaces().clear();
+                nsProperty.getNamespaces().putAll(collectedNamespaces);
+//            }
+//        });
+    }
+
+    private void setSaved(boolean saved) {
+        this.saved = saved;
+    }
+
+    public boolean isSaved() {
+        return saved;
+    }
+    
+    private void setSelectedXpath(String selectedXpath) {
+        this.selectedXpath = selectedXpath;
+    }
+    
+    /**
+     * Utility method for constructing a Document from an XML file.
+     * 
+     * @param file File containing the XML string.
+     * @return a Document instance representing the XML structure.
+     * @throws Exception If an error is encountered while reading the file or parsing its content.
+     */
+    public static Document parseDocument(File file) throws Exception {
+        DocumentBuilderFactory factory = DocumentBuilderFactory.newInstance();
+        DocumentBuilder builder = factory.newDocumentBuilder();
+        return builder.parse(file);
+    }
+    
+    
+    /**
+     * Utility method for adding a child TreeItem into the specified parent TreeItem object.
+     * 
+     * @param parent Parent TreeItem or null if this is root.
+     * @param node DOM Node to be associated with this TreeItem.
+     */
+    private void addTreeItem(TreeItem parent, Node node) {
+        TreeItem childTreeItem;
+        if (null == parent) {
+            childTreeItem = new TreeItem(treeViewWidget, SWT.NONE);
+        } else {
+            childTreeItem = new TreeItem(parent, SWT.NONE);
+        }
+        childTreeItem.setText(node.getNodeName());
+
+        // Select the icon based on the node type.
+/*        if (node.getNodeType() == Node.ELEMENT_NODE) {
```

Comment:
Shall we remove these commented sections

Reply:
Fixed in https://github.com/wso2/integration-studio/pull/1114